example.py

```diff
@@ -1,52 +1,43 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2017 Anidata
```

Comment:
Not important right now, but I think we need to pick a license and stick to it and centralize in the near future.